spot-client/src/spot-tv/app-state/actions.js

```diff
@@ -156,25 +161,21 @@ export function createSpotTVRemoteControlConnection({ pairingCode, retry }) {
                 dispatch(setPermanentPairingCode(''));
 
                 throw error;
-            } else if (retry) {
-                doConnect();
+            } else if (retry || initiallyConnected) {
+                const jitter = getJitterDelay(3);
+
+                logger.log(`Spot TV will try to reconnect after ${jitter}ms`);
+
+                return new Promise((resolve, reject) => {
```

Comment:
What is the promise for?

Reply:
I was trying to keep the method's signature. I see it is used as `catch` in `doConnect` at line 220. I'm not sure about that. Do you think it's safe to remove the promise?

Reply:
I see. You actually read docs like a good person. And the throws are how it "returns" a rejected promise. Looks good as is.